### PR TITLE
Added simple Navigation

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**
Navigation
Web capture_14-2-2022_02852_127 0 0 1

### CODE REVIEW

File: Components-JS/Navigations.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const elem = document.querySelector('.heading1')
+console.log(elem)
```
Comment (sanjitsarkar) [inline Components-JS/Navigations.js:2]: remove debugging statement like console.log in production

File: Components-JS/Navigations.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const elem = document.querySelector('.heading1')
+console.log(elem)
+const observer = new IntersectionObserver(entries =>{
+    console.log("working")
```
Comment (sanjitsarkar) [inline Components-JS/Navigations.js:4]: Here too

File: Components-JS/modal.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const doAction = document.querySelector('#modal-opener');
+const disp = document.querySelector('.modal-box')
+const container = document.querySelector(".container")
+
+//  function lateRun(){
```
Comment (sanjitsarkar) [inline Components-JS/modal.js:5]: Remove unused codes

File: ComponentsCss/code.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    display: flex;
+    flex-direction: column;
+    gap: 0.1rem;
+    
+}
+.line {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    padding: 0.5rem;
```
Comment (sanjitsarkar) [inline ComponentsCss/code.css:27]: don't use rem or em for value lesser than 10px

File: ComponentsCss/code.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+}
+.element {
+    color: #2f6f9f;
+}
+.attribute {
+    color: #006ee0;
+    padding-left: 0.3rem;
+}
+.attr-content {
+    color: #d73038;
```
Comment (sanjitsarkar) [inline ComponentsCss/code.css:46]: use variable for colors

File: Navigations/navigations.css [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+    align-items: center;
+}
+.nav-item.short .frame{
+    display: flex;
+    flex-direction: column;
+    border: solid;
+    height: 80%;
+    width: 80%;
+    overflow-y: scroll;
+    box-shadow: 1rem 1rem 1rem rgb(148, 148, 148);
```
Comment (sanjitsarkar) [inline Navigations/navigations.css:116]: here too no variable for color

File: Navigations/navigations.css [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+    border-radius: 0 0 1rem 0px;
+    color: white;
+    display: flex;
+    justify-content: space-between;
+    background-color: #9400D3;
+    box-shadow: 2px 2px 3px grey;
+}
+.nav-item.short .frame .heading .expand1{
+    /* width: 20%;
+    height: 20%;
```
Comment (sanjitsarkar) [inline Navigations/navigations.css:156]: remove unused commented code

**Review by sanjitsarkar — APPROVED** [review]

**Review by Hamza-777 — APPROVED** [review]
Looks good hrishikesh just make the changes requested by sanjit

**Review by md-amir-gauhar — APPROVED** [review]
Everything looks good. Just take care of the suggestions given above

**Pull request merged**